### DISCUSSION TITLE
How to Create a Modal Dialog Window

### DISCUSSION THREAD

**Discussion by kgday** [lapce/floem#705] [Q&A]
I can see Floem has a lot of potential,  and I am currently evaluating it.

From the examples it is obvious, I can have multiple windows open,  but what if I want to create a customer modal dialog?  For example in a database application, editing the details of an item.

Thanks.

**Comment by jrmoulton** [comment]
You could have a dialog as a separate window or you can use https://docs.rs/floem/latest/floem/action/fn.add_overlay.html

**Comment by kgday** [comment]
Showing the window is no problem, but how to show it modally, ie disable the rest of the application windows?  The overlay is definately an option.  I shall consider that.
